test_connectShieldToTTNgateway.ino

```diff
@@ -0,0 +1,171 @@
+#include <lmic.h>
+#include <hal/hal.h>
+#include <SPI.h>
+
+#define MAX_CHANNELS;
+#define MAX_BANDS;
+#define LIMIT_CHANNELS;
+#define BAND_MILLI;
+#define BAND_CENTI;
+#define BAND_DECI;
+#define BAND_AUX;
+#define CFG_sx1276_radio 1
+//#define LMIC_USE_INTERRUPTS
+#define LMIC_DEBUG_LEVEL 2
+#define USE_ORIGINAL_AES
+//#define USE_IDEETRON_AES
+
+
+//void setup() {
+//  LMIC_setupBand();
+//}
+
+// LoRaWAN NwkSKey, network session key
+static const PROGMEM u1_t NWKSKEY[16] = { 0xC0, 0x3D, 0xBA, 0xDB, 0xBA, 0x96, 0x4E, 0x59, 0x57, 0x5D, 0x7D, 0x4E, 0x99, 0xFF, 0xBB, 0x59 };
```

Comment:
Registered the Node (and an "Application") at TTN to get network session key, application session key and end-device address.